src/sif/dynamiccost.cc

```diff
@@ -348,184 +348,129 @@ void DynamicCost::set_use_living_streets(float use_living_streets) {
              2.f * (1.f - use_living_streets) * (1.f - kMinLivingStreetFactor));
 }
 
-void ParseSharedCostOptions(const rapidjson::Value& value, CostingOptions* pbf_costing_options) {
```

Comment:
i moved the guts of this function into `ParseBaseCostOptions` and in the future i hope to move the guts of most costings' parsing into this function maybe even such that only this function exists